### ISSUE TITLE
About training hyperparameter settings

### ISSUE THREAD

**Issue by qumengxue** [zichengsaber/LAVT-pytorch#4]
Hi,
Thanks for your work! 
However, I encountered some problems in the reproduction.
I use the following command to reproduce the result of RefCOCO with 4 RTX3090: 
_CUDA_VISIBLE_DEVICES=0,1,2,3 python -m torch.distributed.launch --nproc_per_node 4 --master_port 12345 main.py --batch_size 2 --cfg_file configs/swin_base_patch4_window7_224.yaml --size 448_
Here are the results: 
20epoch: **mIOU 61.836 Overall IOU 64.130**
40epoch: **mIOU 66.932 Overall IOU 66.146**
The result is still far from your reported results **mIOU 71.949 Overall IOU 72.246**. 

 I think my equipment and hyperparameter settings are the same as yours, but the results are much worse. Do you use a different hyperparameter or configuration? 

Looking forward to your reply soon! Thanks!

**Comment by rohjunha** [comment]
Thank you for the implementation as well and I also have worse accuracy from my own training.
I have trained a model with 8 A40 GPUs with a larger batch size and I got mIOU 56.88, overall IOU 57.6 at epoch 49.
The training was interrupted a few times, so it had to resume training a few times.

**Comment by Starboy-at-earth** [comment]
@zichengsaber I also get mIOU 60.268 Overall IOU 60.040 on refcoco val set, much more inferior to the reported rereults... Could you please release the ckpt files? Thank you for your contribution!!!
